app/views/webpages/index.html.erb

```diff
@@ -70,34 +79,36 @@
               </p>
       </div>
     </div>
-    <div class="row" style="margin-top: 28px;">
-      <div class="media text-center" style="width: 160px;text-align:center">
-        <a style="width:100%;" href="/book/20000000010359">
+    <div class="row news-item">
+      <div class="media">
+        <a class="news-image" href="/book/20000000010359">
           <%= image_pack_tag "Harrison_0920_border.jpg" %>
         </a>
       </div>
-      <div class="media-body" style="padding-left: 28px;">
-      <h2><a href="/book/20000000010359">Do Right by Me</a>
-            <br><small><em>Learning to Raise Black Children in White Spaces</em></small>
-          </h2>
-          <p class="author">Valerie I. Harrison and Kathryn Peach D'Angelo</p>
-          <p style="line-height:1.2">
-              <strong>LISTEN:</strong>&nbsp;interview&nbsp;on 
-              <a href="http://bit.ly/35E1TLn" target="_blank" rel="noopener"><em>Radio Times</em></a>, 
-              (WHYY, 90.9 FM, Phila) Jan 12<br><strong>EVENTS</strong>: Feb 4 author reading at the 
-              <a href="http://bit.ly/35mCxSg" target="_blank" rel="noopener">FLOP</a>&nbsp;&amp; Feb 8 at 
-              <a href="http://bit.ly/35PZqh7" target="_blank" rel="noopener">Big Blue Marble Bookstore</a>
-              </p>
+      <div class="media-body">
+        <h2>
+          <a href="/book/20000000010359">Do Right by Me</a>
+          <br><small><em>Learning to Raise Black Children in White Spaces</em></small>
+        </h2>
+        <p class="author">Valerie I. Harrison and Kathryn Peach D'Angelo</p>
+        <p style="line-height:1.2">
```

Comment:
@cdoyle-temple I'm surprised to see in-line styles.  Is this temporary?

Reply:
Missed this...